app/economy/management/commands/get_prices.py

```diff
@@ -177,6 +178,57 @@ def cryptocompare():
         refresh_conv_rate(obj.created_on, obj.subscription.token_symbol)
 
 
+def uniswap():
+    """Hangle pulling market data from Uniswap using its subgraph node on mainnet."""
+    endpoint = 'https://api.thegraph.com/subgraphs/name/graphprotocol/uniswap'
+    # Get the top-50 exchange pairs in terms of total ETH locked
+    # See https://github.com/graphprotocol/uniswap-subgraph#important-info
```

Comment:
The bug referred to in that README has now been fixed: https://github.com/graphprotocol/graph-node/issues/892

For an example, here's a query and response for a token that is outside of the top 50. The decimals are handled properly.

```
{
  exchanges(where: { tokenSymbol: "PAN" }) {
    id
    tokenAddress
    tokenSymbol
    tokenName
    tokenBalance
    price
    priceUSD
  }
}
```

```
{
  "data": {
    "exchanges": [
      {
        "id": "0xf53bbfbff01c50f2d42d542b09637dca97935ff7",
        "price": "8526.521918270576287491",
        "priceUSD": "0.015992310639683812",
        "tokenAddress": "0xd56dac73a4d6766464b38ec6d91eb45ce7457c44",
        "tokenBalance": "660725.802210151780432198",
        "tokenName": "Panvala pan",
        "tokenSymbol": "PAN"
      }
    ]
  }
}
```